<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingInterfaceClass</class>
 <widget class="QMainWindow" name="DrawingInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DrawingInterface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>121</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius: 5px;
border: 2px solid black;</string>
    </property>
    <property name="title">
     <string>Colors</string>
    </property>
    <widget class="QCheckBox" name="red">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="green">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: lightgreen;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="blue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: blue;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="cyan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: cyan;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="magenta">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: magenta;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="yellow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: yellow;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="brown">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: brown;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="black">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="darkGreen">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: darkgreen;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="darkBlue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: darkblue;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="gray">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: gray;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="orange">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
border-radius: 5px;
border: 2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QCheckBox" name="eraser">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>eraser.png</normaloff>eraser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>280</y>
      <width>151</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pen width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="widthSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="drawing">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>350</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>drawing</string>
    </property>
   </widget>
   <widget class="QLabel" name="word">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Word:</string>
    </property>
   </widget>
   <widget class="QLabel" name="generatedWord">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(32, 255, 248);
background-color: rgb(29, 251, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
